backend/httpclient/http_client_test.go

```diff
@@ -20,6 +20,7 @@ func TestNewClient(t *testing.T) {
 		client, err := New(Options{
 			Timeouts: &TimeoutOptions{
 				Timeout:               time.Second,
+				DialTimeout:           time.Second,
```

Comment:
Maybe a different value of this one together with a require below would be good just in case.

```suggestion
				DialTimeout:           7 * time.Second,
```